### ISSUE TITLE
Ability to add comment to zone and records

### ISSUE THREAD

**Issue by telefax** [tuxis-ie/nsedit#127]
I'm looking into replacing some other system with pdns and nsedit.
However, the other system has the ability to add comments to zones and records.
For instance, Ticket number containing the order information and maybe which customer who owns this domain. Gets quite messy when a customer has 50 domains seemingly unrelated to their brand =)

Would this be possible with nsedit?
Maybe store this metadata in a separate sqlite database, which can also be exported?
I guess it wouldn't be a good idea to tamper with the pdns backend it self.

Thanks for a great tool btw, best I've found!

**Comment by telefax** [comment]
Interesting, there actually is a "comment" in the stock pdns database that could be used :)

**Comment by tuxis-ie** [comment]
Indeed. Isn't too difficult to implement. However, when would you want to actually see this comment?

**Comment by telefax** [comment]
There is always a risk of cluttering the user interface. I like the way its now, its detailed but rather clean.

Not sure if it would be good to have another fold out when clicking on the actual record, or clicking on some little comment icon or not. At the same time, I really like having comments available all the time when I scroll through records :)

Two possible ways I see:

* Having some sort of checkbox or similar labeled "Show/hide comments" and then you could be able to set your default choice in the config file.
And then show the data in just add another column in the zone/record list.

* Having a little comment icon which indicates that there is a comment, either its hidden when there are no comments or it changes to a specific color depending on state.

Maybe the zone records need to "break out" of the parent folding somehow, since each levels removes some usable screen width. This would make the comment field less intrusive.

**Comment by bjoe2k4** [comment]
+1 for the Icon solution (similar to DNSSEC icon, but not necessarily a new column). And maybe a way to change the comment without updating soa of pdns API allows.
